archetypes/default/plugin.js

```diff
@@ -1,4 +1,6 @@
 /**
+ * Plugin Name
+ * Describe your plugin here
```

Comment:
```suggestion
 * Plugin Name
 *
 * Describe your plugin here
 *
```

Just a random thought. What do you think about having these separated by spaces to be more noticeable?